curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/64440a50d076c204051f2cc0.md

```diff
@@ -1,46 +1,64 @@
 ---
 id: 64440a50d076c204051f2cc0
-title: Step 34
+title: Step 25
 challengeType: 0
-dashedName: step-34
+dashedName: step-25
 ---
 
 # --description--
 
 You need a function to convert view counts to a more readable format. For example, if the view count is `1000`, it should display as `1k` and if the view count is `100,000` it should display as `100k`.
 
-Create an arrow function called `viewCount` with a parameter called `views`.
+Create a `viewCount` function with a `views` parameter. If `views` is greater than or equal to `1000`, return a string with the `views` value divided by `1000` and the letter `k` appended to it. Make sure to round `views / 1000` down to the nearest whole number.
+
+Otherwise, return the `views` value.
```

Comment:
I wonder if it would be beneficial to have some sort of example of the returned output.
Because in the past on the forums and discords, campers going through these exercises always perform better when there is some sort of example output to refer to instead of just the instructions.

maybe something like this

```suggestion
Otherwise, return the `views` value.

For example, if `views` is `1000` your return value should be the string `1k`.
```